app/src/main/res/layout/mediaplayerinfo_activity.xml

```diff
@@ -46,6 +46,25 @@
             tools:background="@android:color/holo_orange_light"
             android:layout_marginBottom="12dp" />
 
+        <TextView
+            android:id="@+id/seek_display"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_above="@id/sbPosition"
+            android:layout_centerInParent="true"
+            android:layout_marginBottom="5dp"
```

Comment:
> It might need a little bit of tweaking since it covers the text just a tad

I would make this margin a bit bigger, so that it "deliberately" covers the title and has a bit more distance to the seek bar (so it is not covered by the thumb). I think covering the title with a temporary view is okay. If we changed the margin of the title to be fully visible, it would make the cover smaller on small screens.

In general, Material Design uses multiples of 8dp (and sometimes 4dp) for spacing. It would be great if you could adapt the numbers accordingly.